week2/who_are_you/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="shortcut icon" href="./favicon.ico" type="image/x-icon" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>덕력 테스트</title>
+  </head>
+  <body>
+    <header>
+      <h1>덕력 TEST</h1>
+    </header>
+    <main>
+      <section class="scoreBoard">
+        <span class="scoreBoard__score">0</span>
+        <span>점</span>
+      </section>
+      <div class="imageBoard">
+        <img src="" alt="너 누구야?"/>
+      </div>
+      <div class="answer">
+        <header class="answer__title">
+          <h2>이 사람은 누구일까요?</h2>
+        </header>
+        <ul class="answer__list">
+          <li>조유리</li>
+          <li>장원영</li>
+          <li>윈터</li>
+          <li>은하</li>
+          <li>태연</li>
+        </ul>
+      </div>
+      <hr />
+      <div class="buttonList">
+        <button class="buttonList__shuffle">
+          <a href="/">다시하기</a>
```

Comment:
```suggestion
다시하기
```
일단 이 a테그 제거하고, 아래 튜토리얼(?)을 따라와주세여